test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -551,7 +551,7 @@ public void GetValidationMetadata_RequiredAttribute_SetsIsRequiredToTrue()
 
             var attributes = new Attribute[] { required };
             var key = ModelMetadataIdentity.ForProperty(typeof(int), "Length", typeof(string));
-            var context = new ValidationMetadataProviderContext(key, attributes);
+            var context = new ValidationMetadataProviderContext(key, new ModelAttributes(attributes, new object[0]));
```

Comment:
looks new and different.  am I correct calls to the 2-parameter `ModelAttributes` constructor are all for property metadata?